tools/Environments/DevHome.Environments/Selectors/CardItemTemplateSelector.cs

```diff
@@ -11,7 +11,7 @@ public class CardItemTemplateSelector : DataTemplateSelector
 {
     public DataTemplate? CreateComputeSystemOperationTemplate { get; set; }
 
-    public DataTemplate? PerProviderComputeSystemTemplate { get; set; }
+    public DataTemplate? ComputeSystemTemplate { get; set; }
 
```

Comment:
PerProviderComputeSystemTemplate ->  ComputeSystemTemplate
These changes are all part of the "Grouping" revert : Environments page recently got a grouping of VMs added to the UI, but this was later disapproved by the designer. These changes are from the revert commit.